pmultiqc/modules/quantms/quantms.py

```diff
@@ -5031,6 +5051,130 @@ def add_sub_section(self, sub_section, plot, order=0, description="", helptext="
             }
         )
 
+    # ProteoBench
+    def draw_proteobench(self, pb_dict):
```

Comment:
why draw? This sound like probabilities to me. It's also hard to assess such long documents. I would go for splitting it up into smaller files. PyLint recommends e.g. to have less then 1000 lines of code per file to keep it organized.